scripts/ci/run-jenkins.sh

```diff
@@ -20,7 +20,7 @@ fi
 
 if [[ ${CI_TAGS} == 'mobile_static' ]]; 
 	then 
-	EXTRA_CONF_FLAGS="${EXTRA_CONF_FLAGS} --with-profile4_x=no --enable-minimal=com,remoting,reflection_emit_save,reflection_emit,appdomains --with-mobile_static --enable-nls=no";
+	EXTRA_CONF_FLAGS="${EXTRA_CONF_FLAGS} --with-runtime_profile=mobile_static";
```

Comment:
Where do we set the --enable-minimal settings now? I don't see it being moved into autoconf/automake.